var/spack/repos/builtin/packages/upcxx/package.py

```diff
@@ -40,12 +23,14 @@ class Upcxx(Package):
 
     homepage = "https://upcxx.lbl.gov"
 
+    version('2019.9.0', '7642877e05300e38f6fa0afbc6062788', default=True)
```

Comment:
`default=True` is not needed here, Spack already defaults to the latest release tarball.

Reply:
I will address this.

- [ ] remove `default=True`